manifests/dashboard-install/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- dashboard-clusterrolebinding.yaml
+- dashboard-clusterrole.yaml
+- dashboard-deployment.yaml
+- dashboard-service.yaml
+- dashboard-serviceaccount.yaml
+images:
+- name: quay.io/argoproj/argo-rollouts-dashboard
```

Comment:
Just curious if the image is available as it seems not exist at https://quay.io/organization/argoproj

Reply:
my mistake, thanks. Updated to the correct one `quay.io/argoproj/kubectl-argo-rollouts`